apps/storybook-react/stories/Button.stories.jsx

```diff
@@ -146,6 +282,28 @@ export const form = () => {
   )
 }
 
+export const fileUpload = () => {
+  return (
+    <Wrapper>
+      <input
+        type="file"
+        id="file-upload"
+        style={{ display: 'none' }}
+        multiple
+      />
+      <label htmlFor="file-upload">
+        <Button as="span">Upload</Button>
```

Comment:
This results in invalid html: 
![image](https://user-images.githubusercontent.com/2081882/85692413-a7ef7980-b6d5-11ea-9c43-5b02857c66f8.png)

Also not possible to use keyboard to focus on the button.